src/_pytest/fixtures.py

```diff
@@ -944,7 +975,11 @@ def _eval_scope_callable(
 
 @final
 class FixtureDef(Generic[FixtureValue]):
-    """A container for a fixture definition."""
+    """A container for a fixture definition.
+
+    Note: At this time, only explicitly documented fields and methods are
```

Comment:
`cache_result` is explicitly documented, but we consider it internal according to the changelog... is this definition incomplete, or am I missing something?

Reply:
I think `cached_result` is not documented, maybe I'm missing something?

Reply:
Hmm yeah sorry, I was not clear and I misunderstood: I thought "documented" meant the comments we leave on the attributes and that would appear in the docs, but that's not the case:

https://docs.pytest.org/en/stable/reference/reference.html#fixturedef

Only `scope` is explicitly documented, sorry for the confusion!